addons/sale_timesheet/data/sale_service_demo.xml

```diff
@@ -1685,5 +1685,23 @@
             <field name="amount">-30.0</field>
         </record>
 
+        <record id="project_update_1" model="project.update" context="{'default_project_id': ref('project.project_project_1')}">
```

Comment:
We have to create a default description based on the context value, thus we have to give this value in the context, otherwise, there won't be any description created.

Sadly I couldn't find an easy other way to deal with it in order to be easily implemented in a UX context and for the data's creation.